frontend/src/components/routingComponents/Routing.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import SignIn from '../../pages/SignIn';
+import SignUp from '../../pages/SignUp';
+import Home from '../../pages/Home';
+import Topics from '../../pages/Topics';
+import MyTeam from '../../pages/MyTeam';
+import Profile from '../../pages/Profile';
+import Nav from './Nav';
+import ProtectedRoute from './ProtectedRoute';
+import { setAuthenticationStatus } from '../../state/actions';
+
+const Routing = () => {
+  const [isAuthenticated, setAuthentificationStatus] = useState(false);
+
+  const dispatch = useDispatch();
+
+  const changeAuthentificationStatus = () => {
+    dispatch(setAuthenticationStatus(!isAuthenticated));
```

Comment:
shouldn't it be `setAuthneticationStatus(true)` ?

Reply:
can there be situations when you can login twice and due to this function, you accidentally logout?

Reply:
fixed